### ISSUE TITLE
Skip link isn't focused when navigating from profile menu

### ISSUE THREAD

**Issue by aitchiss** [forem/forem#14496]
**Describe the bug**

When navigating by keyboard, if I select a link from the right-hand profile nav menu, the profile menu button is refocused after the route change and not the skip link.

**To Reproduce**

- Press the `Tab` key until the profile menu is focused
- Press `Enter` to open it, then `Tab`/`Enter` to choose a link from the menu
- Notice that when the page changes, the profile menu button is still focused and not the skip link

**Expected behavior**

- When changing route using the keyboard, the skip link should be focused immediately on the new page

**Screenshots**

https://user-images.githubusercontent.com/20773163/129367969-7b7ad8f7-7267-4312-bb72-faae2807778c.mp4

**Desktop (please complete the following information):**

- OS, version: MacOS Catalina
- Browser, version: Safari, Chrome

**Additional context**

We have code to manage focus in/out of the profile menu - when staying on the same page, a dropdown button _should_ regain focus when the dropdown closes. This is a bit of a special case because the profile menu button doesn't change between routes.

**Comment by CatalinCsnMaster** [comment]
Hello! can I take this issue? (•‿•)

**Comment by nickytonline** [comment]
For sure @CatalinCsnMaster! Assigned!

**Comment by aitchiss** [comment]
I think this might have been closed by mistake @nickytonline 🤔  Thanks for taking it on @CatalinCsnMaster!

**Comment by nickytonline** [comment]
Oops! Thanks for reopening @aitchiss!

**Comment by Link2Twenty** [comment]
It seems to be quite hit or miss for me, sometimes the skip appears and at other times it doesn't. I couldn't see and real pattern.
